playbooks/roles/start_scenario/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+csm_home: '/home/linux/customer-service-monitoring'
```

Comment:
why this block duplicated in inventory?

Reply:
Replaced with required var list in `defaults` for each role